src/utils/apiCall/getData.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+export const getData = async (request, url, dispatch) => {
+  try {
+    const res = await axios({
+      method: request,
+      url: url,
+    });
+    if (res.status === 200) {
```

Comment:
suggestion: check for both status 200 and 201 as mockbee uses them interchangeably.